### DISCUSSION TITLE
Custom point cloud dataset classification and segmentation

### DISCUSSION THREAD

**Discussion by narges-tk** [pyg-team/pytorch_geometric#8545] [Q&A]
Thanks for this nice repository! 
Could you please guide me on how I can use classification and segmentation networks like pointcnn on a customized point cloud dataset?
Thanks in advance.

**Comment by xnuohz** [comment]
For point cloud benchmark, see:
https://github.com/pyg-team/pytorch_geometric/tree/master/benchmark/points
For some examples, see:
https://github.com/pyg-team/pytorch_geometric/blob/master/examples/pointnet2_segmentation.py
https://github.com/pyg-team/pytorch_geometric/blob/master/examples/pointnet2_classification.py
